admin_guide/metrics_recommendations.adoc

```diff
@@ -0,0 +1,30 @@
+== OpenShift v3.5 metrics recommendations
+
+In OpenShift v3.4 metrics https://docs.openshift.org/latest/install_config/cluster_metrics.html[documentation]
+are presented results which changed to better in OpenShift v3.5 metrics
+
+The reason for better results lies in change of `METRIC_RESOLUTION` parameter
```

Comment:
Suggestion:  "in the change of the default value of the METRICS_RESOLUTION parameter